docs_user/modules/con_fips-support.adoc

```diff
@@ -0,0 +1,11 @@
+[id="fips-support_{context}"]
+
+= FIPS support
+
+If you are adopting an {OpenStackShort} {rhos_prev_ver} FIPS environment to {OpenStackShort} {rhos_curr_ver}, your adopted cluster will remain a FIPS cluster. There is no option to change FIPS status during adoption.
```

Comment:
```suggestion
If you are adopting a {rhos_prev_long} {rhos_prev_ver} FIPS environment to {rhos_long} {rhos_curr_ver}, your adopted cluster remains a FIPS cluster. There is no option to change FIPS status during adoption.
```
The first part without the attributes translates to "If you are adopting a Red Hat OpenStack Platform 17.1 FIPS environment to Red Hat OpenStack Services on OpenShift (RHOSO) 18.0..." Is that what you intended?